react-native-libraries.json

```diff
@@ -3280,9 +3281,15 @@
     "android": true,
     "web": false,
     "expo": true,
-    "windows": false,
+    "windows": true,
     "unmaintained": false
   },
+  {
+    "githubUrl": "https://github.com/microsoft/react-native-dualscreen",
+    "android": true,
+    "windows": true,
+    "npmPkg": "https://www.npmjs.com/package/react-native-dualscreeninfo"
```

Comment:
Oh, I have just spotted that `npmPkg` here should be just the package name (`react-native-dualscreeninfo`).